docs/developer-docs/devquests.md

```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
 4. Homebrew that is essentially the same as another you've done will not count (i.e. multiple Flappy Bird clones can only garner credit once)
 5. One set per large homebrew collection maximum (i.e. One Net Yaroze 2014 Collection set allowed)
 
+### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)
+
+![DQ022](https://retroachievements.org/Images/102134.png)
+
+**Type**: Revision, Maintenance 
+**Goal**: Obtain 12 Points by dusting off older sets, bringing them up to current RA standards  
+**Repeatable**: No
+
+1. The Quest is completed upon obtaining 12 points towards it. Full Set Dust-Off = 4 Points, Misc Task = 1 Point, at least one set dust-off required for the badge.
```

Comment:
Consider making the point descriptions sub-bullets for more visibility.

Reply:
Done